code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -2478,3 +2478,11 @@
 	icon_state = "tatortot"
 	list_reagents = list("nutriment" = 4)
 	filling_color = "FFD700"
+
+/obj/item/weapon/reagent_containers/food/snacks/cucumberslice
+	name = "Cucumber Slice"
+	desc = "A slice of cucumber"
+	icon_state = "cucumberslice"
+	list_reagents = list("nutriment" = 1)
```

Comment:
> Carnivores: Unathi, Slime People
Omnivores: Human, Tajaran, Vox, Shadowpeople, Golems, Plasmaman, Nucleation, Vox Armalis
Herbivores: Skrell, Kidan, Grey, Wryn
No Diet (null dietflag): Diona, IPC

I think my main concern there was that Vulpkanin are not on the list (maybe because they didn't exist) and therefore I don't know _what_ they are. But I also did not know this existed. Now I do, thanks